datahub-web-react/src/app/preview/DefaultPreviewCard.tsx

```diff
@@ -1,21 +1,88 @@
-import { Space } from 'antd';
+import { Avatar, Divider, Image, Row, Space, Tag, Tooltip, Typography } from 'antd';
 import React from 'react';
 import { Link } from 'react-router-dom';
+import { EntityType } from '../../types.generated';
+import defaultAvatar from '../../images/default_avatar.png';
+import { useEntityRegistry } from '../useEntityRegistry';
 
-interface Props extends React.PropsWithChildren<any> {
-    title?: React.ReactNode;
+interface Props {
+    name: string;
+    logoUrl?: string;
     url: string;
+    description: string;
+    type: string;
+    platform: string;
+    qualifier?: string | null;
+    tags: Array<string>;
+    owners: Array<{ urn: string; name?: string; photoUrl?: string }>;
 }
 
-export default function DefaultPreviewCard({ title, url, children }: Props) {
+const styles = {
+    row: { width: '100%' },
+    leftColumn: { maxWidth: '75%' },
+    rightColumn: { maxWidth: '25%' },
+    logoImage: { width: '48px' },
+    name: { color: '#214F55', fontSize: '18px' },
+    typeName: { color: '#585858' },
+    platformName: { color: '#585858' },
+    ownedBy: { color: '#585858' },
+};
```

Comment:
should we be using styled-components here?

Reply:
Can I style custom components using styled? Eg antd components?